src/Dapper.FSharp/MSSQL.fs

```diff
@@ -168,10 +175,10 @@ module private Evaluators =
 
 [<AbstractClass;Sealed>]
 type Deconstructor =
-    static member select<'a> (q:SelectQuery) = q |> GenericDeconstructor.select1<'a> Evaluators.evalSelectQuery
-    static member select<'a,'b> (q:SelectQuery) = q |> GenericDeconstructor.select2<'a,'b> Evaluators.evalSelectQuery
+    static member select<'a> (q:SelectQuery)       = q |> GenericDeconstructor.select1<'a> Evaluators.evalSelectQuery
```

Comment:
Please, don't align the code like Excel. :) 
See: https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting
and: https://caml.inria.fr/resources/doc/guides/guidelines.en.html